### PR TITLE
[Style] Update skill family page layouts

### LINKED ISSUE #11477: ✨ Admin / Skill families / mostly Styling update
## ✨ Feature

Update the skill families page to the new admin design patterns

### Out of scope for this issue (don't build this even if it's in the design)
- [ ] Moving to the new nav
- [ ] Updates to the table component
- [ ] The "advanced tools" tab and everything in it
- [ ] Search/filter dialog up top (don't worry about it)

## 🎨 Design

[Figma files](https://www.figma.com/design/W2cVcRK5YtrllE0sbVgwEV/Admin---Skills-families?node-id=14051-2833&t=gOuxokYXQXHWhokM-1) by @Jerryescandon 

## ✅ Acceptance Criteria

- [ ] Skill families page (/admin/settings/skill-families)
  - [ ] Header  (background, H1 & breadcrumb)
  - [ ] H2 and content styling
  - [ ] Remove the "edit" link column from the table and add links on the "name" column to bring users to the "information" page
- [ ] Create skill family (/admin/settings/skill-families/create)
  - [ ] Header  (background, H1 & breadcrumb)
  - [ ] H2 and content styling

- [ ] New page - Skill families information (/admin/skill-families/:id:)
  - [ ] Header  (background, H1 & breadcrumb)
  - [ ] H2 and content styling
  - [ ] Only one tab for now
  - [ ] Inline form for displaying information and editing
- [ ] Edit skill family page (/admin/skill-families/:id:/edit)
  - [ ] (should use same component as Create Skill families page)

- [ ] mobile view for all pages

- [ ] Design Review by @Jerryescandon 

## Blockers
```[tasklist]
### Blocked by
- [ ] #11344 
```

### PR DESCRIPTION
**Pull request by esizer**
🤖 Resolves #11477 

## 👋 Introduction

Updates the layouts for the skill family pages.

## 🧪 Testing

1. Build `pnpm run dev:fresh`
2. Login as admin `admin@test.com`
3. Navigate to the skill families pages `/admin/settings/skill-families`
4. Confirm the pages match the mock ups

### CODE REVIEW

File: apps/web/src/pages/SkillFamilies/UpdateSkillFamilyPage.tsx
Comment (petertgiles) [inline apps/web/src/pages/SkillFamilies/UpdateSkillFamilyPage.tsx:1]: Key field should be visible on this page.

Comment (petertgiles) [inline apps/web/src/pages/SkillFamilies/UpdateSkillFamilyPage.tsx:1]: I think the app should navigate to the view skill family page after submission

File: apps/web/src/pages/SkillFamilies/CreateSkillFamilyPage.tsx
Comment (petertgiles) [inline apps/web/src/pages/SkillFamilies/CreateSkillFamilyPage.tsx:1]: I think the app should navigate to the view skill family page after successful submission

File: apps/web/src/pages/SkillFamilies/UpdateSkillFamilyPage.tsx
Comment (petertgiles) [inline apps/web/src/pages/SkillFamilies/UpdateSkillFamilyPage.tsx:1]: Title should be centred and form buttons should be centred in a column in mobile view

**Review by petertgiles — CHANGES_REQUESTED** [review]
Very nice!  I flagged a few things that came up in my reviews here.

File: apps/web/src/pages/SkillFamilies/components/SkillFamilyTable.tsx
[the remark this answers is not in the document]
Comment (esizer) [reply apps/web/src/pages/SkillFamilies/components/SkillFamilyTable.tsx:1]: Fixed in [update search input label](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11926/commits/7bcd159a5ff08e4b01f56b5ee7b08bc0d1af2762)

File: apps/web/src/pages/SkillFamilies/ViewSkillFamilyPage.tsx
[the remark this answers is not in the document]
Comment (esizer) [reply apps/web/src/pages/SkillFamilies/ViewSkillFamilyPage.tsx:1]: Completed all view related changes in [fix view page layout issues](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11926/commits/6290c79e8ef6507d347f12229fbf44f1ad0dcb56)

**Comment by petertgiles** [comment]
I think everything is done except for centering the headings in mobile width

**Comment by esizer** [comment]
> I think everything is done except for centering the headings in mobile width

I think I misenterpreted what you meant by section headings. I did some centering, but was it the wrong heading?

**Comment by esizer** [comment]
> I think everything is done except for centering the headings in mobile width

Thanks for helping me out, it should be fine now after [fix heading centering](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11926/commits/6f6153abf8ff7b9b4afbfa85bd4d4927911b88b7)

**Comment by petertgiles** [comment]
Headings look good!  I noticed one last thing: the gap in the view page is very large.  Could you please double-check?
image

**Comment by esizer** [comment]
> Headings look good! I noticed one last thing: the gap in the view page is very large. Could you please double-check?

Good catch, I missed zeroiing out the margin on that heading 😬 

[fix heading spacing](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11926/commits/08266ba2e33c95d2ee2bb568bf71a0cbaaa84183)

**Review by petertgiles — APPROVED** [review]
Nice!

**Pull request merged**